discovery/service_test.go

```diff
@@ -828,3 +828,85 @@ func TestOrphanSignatureAnnouncement(t *testing.T) {
 		t.Fatal("wrong number of objects in storage")
 	}
 }
+
+// We should ignore node announcements of nodes not associated with any
```

Comment:
Comment should be prefixed with `TestOrphanNodeAnnouncement`. 

Also I'm not sure that this is the correct package for this test. Correct me if I'm wrong, but the underlying logic of ignoring a node announcement if there aren't any matching channels for it is implemented in the router itself.